apps/spark/src/main/java/com/linkedin/openhouse/jobs/util/SparkCatalogUtils.java

```diff
@@ -0,0 +1,32 @@
+package com.linkedin.openhouse.jobs.util;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.iceberg.CatalogUtil;
+import org.apache.iceberg.catalog.Catalog;
+import org.apache.spark.sql.SparkSession;
+import scala.collection.JavaConverters;
+
+public class SparkCatalogUtils {
+  private SparkCatalogUtils() {
+    // utils ctor noop
+  }
+
+  public static Catalog getIcebergCatalog(SparkSession spark, String catName) {
```

Comment:
should this method be still called getCatalog?

Reply:
Can this be added to `tables-test-fixtures` given that the current scope is testing only? Can be exposed for client testing as well.

Reply:
Instead of creating a new utils class in jobs side, I am moving this into spark-runtime. Please let me know if this looks good.